src/test/resources/data/StudentProfilePageUiTest.json

```diff
@@ -37,13 +37,43 @@
       "isInstructor": false,
       "email": "SHomeUiT.benny.c@gmail.tmt",
       "institute": "TEAMMATES Test Institute 1"
+    },
+    "instructor1OfTestingSanitizationCourse": {
```

Comment:
I am fine with putting "instructor", but "instructor1OfTestingSanitizationCourse" is more like an instructor in a specific course. In addition, you also name an instructor as "instructor1OfTestingSanitizationCourse" in the "instructors" bundles.